p5/sketch/Vispy2DRenderer/renderer2d.py

```diff
@@ -372,56 +371,3 @@ def arc(self, *args):
         mode = args[4]
 
         self.render_shape(Arc(center, dim, start_angle, stop_angle, mode))
-
-    def shape(self, vertices, contours, shape_type, *args):
-        """Render a Pshape"""
-        self.render_shape(PShape(vertices=vertices, contours=contours, shape_type=shape_type))
```

Comment:
The 2D renderer calls `render_shape` instead of `render` because `shape.children` needs to be separately rendered. Maybe it's ok to leave `shape` in separate renderers with separate implementation for now?

Reply:
Yes, nice catch. Thanks